docs/website/components/Content.vue

```diff
@@ -37,3 +37,9 @@ export default {
   }
 }
 </script>
+
+<style>
+article.c-rich-text ul {
+  @apply ml-6;
```

Comment:
This is a pretty nasty hack that I'd rather not have to maintain. Not a blocker, but I'd like to dig into why we need to use `outside`. Using `inside` fixes this everywhere. I will follow up and try to rid us of this.

Reply:
I think your fix is in now, so I'll remove this.